syntax/arm64asm.vim

```diff
@@ -86,8 +86,8 @@ syntax match AArch64Modifier /:gottprel_lo12:/ contained
 syntax match AArch64Modifier /:gottprel_g1:/ contained
 syntax match AArch64Modifier /:gottprel_g0_nc:/ contained
 
-syntax match AArch64Modifier /@PAGE/ contained
-syntax match AArch64Modifier /@PAGEOFF/ contained
+syntax match AArch64Modifier /@PAGE/
+syntax match AArch64Modifier /@PAGEOFF/
```

Comment:
The reason that they are marked contained is because they are part of a symbol reference. You cannot simply use `@PAGE`. The `contained` was to indicate that this must be within another match. I think that the problem likely is with the way that symbols are being identified and might be a single letter thing.